### ISSUE TITLE
fatal error: A process has ended with a probable error condition: process ended with exit code 130.

### ISSUE THREAD

**Issue by Brottweiler** [barneygale/mark2#85]
When starting the server, it just crashes, and I can attach it but its not actually running. It's impossible to quit mark2 in any other  way than to SIGKILL it.
It will not respond to anything. ~stop or ~kill does nothing at all. You need to SIGKILL it.

This have happened twice, for two different servers, where the servers are running forge.
Starting the server with "mark2 start aether/" it starts the server as normal, as you'd except, with this at the end:

```
[Minecraft-Server] Done (6.708s)! For help, type "help" or "?"`
[Minecraft-Server] /xxx.xxx.xx.xx:35620 lost connection
```

That last line is showned when both servers gets started. The other where the same thing happens ran Invasion, this one Aether, both using forge.

When attaching;

```
[Minecraft-Server] Done (6.708s)! For help, type "help" or "?"
[Minecraft-Server] /xxx.xxx.xx.xx:35620 lost connection
mark2 stopped.
stopping minecraft server
[Minecraft-Server] Stopping server
fatal error: A process has ended with a probable error condition: process ended with exit code 130.
```

/tmp/mark2/aether.log shows this: https://gist.github.com/Brottweiler/37c9049b6a0fb012338d

It seems to only happen with servers running Forge?
I talked about this on IRC, but I dont know what your plan is.

Edit: This exact same thing happens if you start a server which port is already in use. You have to start it to generate a server.properties file, then SIGKILL it, then change the port, then start it again.

**Comment by Algent** [comment]
I just updated mark2 and I get the same error but not with forge:

When I do "mark2 start" to start mark2, the server run fine as long as I don't touch my ssh windows again.
if I exit it or try to exit the "tail -f" like effect (either by closing putty or just doing ctrl+c like I used to)  it close the mc server instantly but keep mark2 open.
Then if I then try to attach to it I see this exit code 130.

Any idea on how I could solve this ? This is a really big issue for me :|.

Edit: managed to bypass/avoid crash by using -i on start. Still interested in how to fix that.
